CDP4Orm/Dao/Supplemental/PersonDao.cs

```diff
@@ -137,6 +142,59 @@ public string GivenName(NpgsqlTransaction transaction, Guid personIid)
             return personObject != null ? personObject.GivenName : null;
         }
 
+        /// <summary>
+        /// Extracts the salt property from the hstore in the <see cref="Person"/> beiung updated and adds it to the the valueType dictionary
+        /// </summary>
+        /// <param name="transaction">
+        /// The current transaction to the database.
+        /// </param>
+        /// <param name="partition">
+        /// The database partition (schema) where the requested resource will be stored.
+        /// </param>
+        /// <param name="person">The relevant <see cref="Person"/></param>
+        /// <param name="valueTypeDictionaryAdditions">
+        /// This dictionary instance can be used to add additional value (non reference) variables that are to be persisted as a HSTORE implementation together with the supplied <see cref="Thing"/> instance.
+        /// Developers are required to take care to add property keys that are not already in the ValueTypeDictionary that is managed in the respective generated partial class.
+        /// The supplied values will be persisted as is and thus must be in a valid format (escaped) that can be persisted to SQL.
+        /// Even though the additional stored variables are read from the data-store, developers will have to add custom DTO mapping to get retrieve the value again.
+        /// </param>
+        private void ExtractExistingSaltToValueDictionary(NpgsqlTransaction transaction, string partition, Person person, Dictionary<string, string> valueTypeDictionaryAdditions)
+        {
+            using (var command = new NpgsqlCommand())
+            {
+                var sqlBuilder = new System.Text.StringBuilder();
+                sqlBuilder.AppendFormat("SELECT * FROM \"{0}\".\"Person_View\"", partition);
```

Comment:
In general I think too much data is retrieved. SELECT * could be replaced by SELECT /"ValueTypeSet/".
I also think that it should throw an exception when the Person is null.
It's not about the performance, but more about the expectations of the method (by its name) versus what it really does.